helm/charts/aleph/templates/api.yaml

```diff
@@ -129,6 +138,10 @@ spec:
           secret:
             secretName: service-account-aleph
         {{ end }}
+        {{ if .Values.global.prometheus.enabled }}
+        - name: prometheus-multiproc-dir
+          emptyDir: {}
+        {{ end }}
```

Comment:
One thing I’m unsure about: The multiprocess mode collects metrics of multiple processes in file-based databases in this directory. The directory should be emptied on restart, but if we collect a lot of metrics and the processes run for a long time, could we run into a situation where we fill up all available disk space?

Reply:
The underlying storage for emptyDir will come from the Node itself. You can add a sizeLimit if you want to prevent it  from being able to get too big. To help the Scheduler insure a Node has sufficient space, you can specify limits / requests for ephemeral-storage.

Reply:
So I confirmed my assumption manually, everything seems to be working as expected:

* Set a very small size limit of 50 Ki for `prometheus-multiproc-dir`.
* Ran Aleph in a local Kubernetes cluster with Prometheus enabled.
* Navigated to a few routes in the API to trigger metrics collection.
* With some delay, the container was stopped and the pod replaced by a new one. Could confirm this by looking at the k8s events:

  ```
  Warning  Evicted    5m31s  kubelet            Usage of EmptyDir volume "prometheus-multiproc-dir" exceeds the limit "50Ki".
  Normal   Killing    5m31s  kubelet            Stopping container aleph
  ```